lib/ncm-ng-adapter.js

```
@@ -0,0 +1,1029 @@
+  }
+  
+  // If something went wrong, return all packages
+  return packages
+}
+
+/**
+ * Check if we're running in self-certification mode (ncm-cli certifying itself)
+ * @param {string} dir Directory to check
+ * @returns {boolean} True if self-certification detected
+ */
+function checkSelfCertification(dir) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'checkSelfCertification'. Consider to delete it or rename it.

'checkSelfCertification' is defined here:


(lint/suspicious/noRedeclare)